src/main/java/org/primefaces/component/timeline/TimelineBase.java

```diff
@@ -50,13 +54,18 @@
         browserTimeZone,
         height,
         minHeight,
+        maxHeight,
         width,
         responsive,
-        axisOnTop,
-        dragAreaWidth,
+        orientationAxis,
+        orientationItem,
         editable,
+        editableAdd,
+        editableRemove,
+        editableGroup,
+        editableTime,
+        editableOverrideItems,
         selectable,
```

Comment:
Need to update /docs/7.1/components/timeline to add and remove all these properties.

Reply:
By now I don't remove from taglib to mantain compatibility with existing xhtml code, so I only removed these from `TimelineBase`. I document on taglib that the properties was deprecated.

Reply:
That is good but the actual Markdown doc need to be update in the /docs/7.1/ folder.

Reply:
> That is good but the actual Markdown doc need to be update in the /docs/7.1/ folder.

That's a new world for me haha. Check this new commit.